src/main.js

```diff
@@ -1,14 +1,28 @@
 'use strict';
 
 const API_KEY = 'e9979803c3894fa182b399c482279297';
-const HOST_URL = 'https://newsapi.org/v2';
-const SOURCES_PARAM = '/top-headlines';
-
+const HOST_URL = 'https://newsapi.org/v2/';
+const SOURCES_PARAM = 'top-headlines';
 const url = `${HOST_URL}${SOURCES_PARAM}?country=us&apiKey=${API_KEY}`;
 const req = new Request(url);
+const container = document.getElementById('container');
+
 fetch(req)
-  //.then(response => response.json()).then(myJson => console.log(JSON.stringify(myJson)))
-   .then(response => {
-   console.log(response.json());
-   })
-  .catch(error => console.log(error) );
+  .then(response => response.json())
+  .then(response => {
+    console.log(response);
+    for(let i = 0; i < response.articles.length; i++ ){
+      let worldEvent = document.createElement('a');
```

Comment:
Please, use appropriate variable naming. It should not relate with data itself.
For DOM elements you can use `El` postfix. For ex., `dataItemEl`.

Reply:
Variable was renamed. 
Markup wrapper was changed on `article`.